core/testcontainers/core/utils.py

```diff
@@ -53,6 +54,43 @@ def inside_container() -> bool:
     return os.path.exists("/.dockerenv")
 
 
+def create_connection_string(
+    dialect: str,
```

Comment:
If this would be DB API only, I would say "dialect" is fine, but if this should also be used to generate http URLs, I would call this base_scheme.